input/pagecontent/reference.md

```diff
@@ -667,9 +669,13 @@ In the following, we give details and examples of assignments involving various
 
 ##### Assignments with the Coding Data Type
 
-FSH provides a shortcut for setting several of the most common properties of a Coding simultaneously, namely, system, version, code, and display. The syntax is:
+A FHIR Coding has five attributes (system, version, code, display, and userSelected). The first four of these can be set with a single assignment statement. The syntax is:
+
+<pre><code>&lt;Coding&gt; = <i>{CodeSystem name|id|url}|{version string}</i>#{code} <i>"{display string}"</i></code></pre>
 
-<pre><code>&lt;Coding&gt; = {CodeSystem name|id|url}<i>|{version string}</i>#{code} <i>"{display string}"</i></code></pre>
+The only required part of this statement is the code (including the # sign), although it is rare to have a Coding without a code system. The version string cannot appear without a code system.
+
+Whenever this type of rule is applied, whatever is on the right side **entirely replaces** the previous value of the Coding on the left side. For example, if a Coding has a value that includes a display string, and a new assignment is made that replaces the system and code but has no display string, then result is a Coding without a display string.
```

Comment:
typo: "**then** result is a Coding without a display string"

Reply:
fixed